src/main.cpp

```diff
@@ -561,6 +564,22 @@ int main( int argc, char *argv[] )
     get_options().load();
     set_language();
 
+#ifdef TILES
+    SDL_version compiled;
+    SDL_VERSION( &compiled );
+    DebugLog( D_INFO, D_SDL ) << "SDL version used during compile is "
+                              << static_cast<int>( compiled.major ) << "."
+                              << static_cast<int>( compiled.minor ) << "."
+                              << static_cast<int>( compiled.patch );
+
+    SDL_version linked;
+    SDL_GetVersion( &linked );
+    DebugLog( D_INFO, D_SDL ) << "SDL version used during linking is "
```

Comment:
Is this also the SDL version when it's dynamically loaded?

Reply:
Yes.

Reply:
Should probably make it clear in the message then.

Reply:
Look if that's better?